src/components/Button/Button.scss

```diff
@@ -150,6 +150,11 @@ $iconWidth: 16px;
         &_action {
             --yc-button-background-color: var(--g-color-base-brand);
             --yc-button-background-color-hover: var(--g-color-base-brand-hover);
+            --yc-button-outline-color: var(--g-color-base-brand);
+
+            &:focus::before {
+                outline-offset: 1px;
```

Comment:
Should be reset back to 0 when `selected` state is active

Reply:
Changed my mind. Let's add offset and color for selected state too

Reply:
I think one shouldn't because the focus will "jump" when selected state will change